internal/envoy/bootstrap.go

```diff
@@ -27,11 +30,25 @@ import (
 	envoy_api_v2_core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 	bootstrap "github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2"
 	matcher "github.com/envoyproxy/go-control-plane/envoy/type/matcher"
+	"github.com/gogo/protobuf/proto"
```

Comment:
```suggestion
"github.com/google/protobuf/proto"
```

Reply:
Oops, sorry!  Apparently now I have to deal with vscode autocorrecting and adding stuff like this, just like my phone! ;)  This should have been `github.com/golang/protobuf/proto` instead.